.github/ISSUE_TEMPLATE/story.md

```diff
@@ -46,7 +46,7 @@ What else should contributors [keep in mind](https://fleetdm.com/handbook/compan
 - [ ] REST API changes: TODO <!-- Specify changes in the the REST API doc page as a PR to reference docs release branch following the guidelines in the handbook here: https://fleetdm.com/handbook/product-design#drafting Put "No changes" if there are no changes necessary. Move this item to the engineering list below if engineering will design the API changes. -->
 - [ ] Fleet's agent (fleetd) changes: TODO <!-- Specify changes to fleetd. If the change requires a new Fleet (server) version, consider specifying to only enable this change in new Fleet versions. Put "No changes" if there are no changes necessary. -->
 - [ ] GitOps changes: TODO <!-- Specify UI changes for read-only GitOps mode. Specify changes to fleetctl generate-gitops. Put "No changes" if there are no changes necessary. -->
-- [ ] Activity changes: TODO <!-- Specify changes to the Audit log page in the contributor docs. Put "No changes" if there are no changes necessary. -->
+- [ ] Activity changes: TODO <!-- Specify changes as a closed (unmerged) PR to the auto-generated Audit log page in the contributor docs. Put "No changes" if there are no changes necessary. -->
```

Comment:
@noahtalerman what do you think of this? We make a regular PR, so reviews will still be auto-requested, but then close it after approving instead of merging. (IIRC we started doing regular PRs because it was easy to forget to manually request reviews on draft PRs.)

Reply:
> but then close it after approving instead of merging

The Product Designer wouldn't close it though right? Sounds like you (API design DRI) would close it when it's approved.

If that's right, then the story template might not need to change 🤷

Reply:
@marko-lisica that's a good point! This may be more of a handbook change